kolibri/plugins/management/templates/management/management.html

```diff
@@ -1,4 +1,19 @@
-{% extends "kolibri/base.html" %}
-{% block content %}
-<span>The management app!</span>
-{% endblock %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kolibri</title>
+  </head>
+  <body>
+    <main v-ref:main></main>
```

Comment:
That makes sense. Is it actually used though? If not and it should be left in, let's add a comment saying not to delete it as dead code.